docs/source/example_input/boosted_frame_script.py

```diff
@@ -175,21 +182,21 @@ def dens_func( z, r ):
            zf=zfoc, gamma_boost=boost.gamma0 )
 
     # Convert parameter to boosted frame
-    v_window, = boost.velocity( [ v_window ] )
+    v_window_boosted, = boost.velocity( [ v_window ] )
     # Configure the moving window
-    sim.set_moving_window( v=v_window )
+    sim.set_moving_window( v=v_window_boosted )
 
     # Add a field diagnostic
     sim.diags = [ FieldDiagnostic(diag_period, sim.fld, sim.comm ),
                   ParticleDiagnostic(diag_period,
                     {"electrons":sim.ptcl[0], "bunch":sim.ptcl[2]}, sim.comm),
-                  BoostedFieldDiagnostic( zmin, zmax, c, dt_snapshot_lab,
+                  BoostedFieldDiagnostic( zmin, zmax, v_window, dt_snapshot_lab,
                     Ntot_snapshot_lab, boost.gamma0, fieldtypes=['rho','E','B'],
-                    period=diag_period, fldobject=sim.fld, comm=sim.comm),
-                  BoostedParticleDiagnostic( zmin, zmax, c, dt_snapshot_lab,
-                    Ntot_snapshot_lab, boost.gamma0, diag_period, sim.fld,
-                    select={'uz':[0.,None]}, species={'electrons':sim.ptcl[2]},
-                    comm=sim.comm )
+                    period=record_period, fldobject=sim.fld, comm=sim.comm),
```

Comment:
Here, I would go maybe with `write_period` instead of `record_period` as it more explicitly tells the user that something is written to disk. However, I am open to any other suggestions.

Reply:
Thinking further about this, `record_period` is maybe also okay as in the OpenPMD standard, written outputs are also called 'records'.

Reply:
I think `write_period` is fine and its clearer how it works this way